PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-detail/pokemon-detail.component.html

```diff
@@ -645,12 +645,26 @@ <h2 name="abilities">Abilities</h2>
         </table>
 
         <h2 name="spawns">Spawns</h2>
-        <div *ngIf="model.pokemon.spawns.length === 0" class="no-table-container-outer">
+        <div *ngIf="model.visibleSpawns.length > 0">
+            <div *ngIf="!model.areOnlyEventExclusiveSpawnsAvailable">
+                <button *ngIf="!model.areEventExclusiveSpawnsHidden" type="button" (click)="hideEventExclusiveSpawns()">
+                    Hide inactive event-exclusive spawns
+                </button>
+                <button *ngIf="model.areEventExclusiveSpawnsHidden" type="button" (click)="showEventExclusiveSpawns()">
+                    Show inactive event-exclusive spawns
+                </button>
+            </div>
+            <div *ngIf="model.areOnlyEventExclusiveSpawnsAvailable">
+                <p class="empty">This Pokémon is only obtainable while events are running in PokéOne, thus showing event-exclusive spawns.</p>
```

Comment:
Change message to "This Pokémon was only available during events in the past". Change font color to red (color: var(--text-accent))